orbit/db2.py

```diff
@@ -71,20 +86,7 @@ class NewUser(BaseModel):
     password = CharField()
 
 
-# TODO move this information to some config file
-def init_db():
-    DB.create_tables([User, Session, Submission, Assignment, NewUser])
-    ASSIGNMENTS = [
-        ("setup", "introductions"),
-        ("E0", "exercise0"),
-        ("E1", "exercise1"),
-        ("E2", "exercise2"),
-        ("P0", "programming0"),
-        ("P1", "programming1"),
-        ("P2", "programming2"),
-        ("F0", "final0"),
-        ("F1", "final1"),
-    ]
-    Assignment.insert_many(
-        ASSIGNMENTS, fields=(Assignment.web_id, Assignment.email_id)
-    ).execute()
+# automatically create tables
+# this actually does IO, creates orbit.db calls CREATE TABLE, etc.
+# maybe this should move to some initializer function
```

Comment:
perhaps it could be called during the build process so that an empty db exists in the container image instead of deferring to runtime.

Reply:
I liked this line because it automatically makes the db when working locally, though I agree it's out of place. I'm not sure a nice way to replicate that